src/utils/parseGlobalAccessToken.ts

```diff
@@ -15,8 +14,8 @@ const parseGlobalAccessToken: ParseGlobalAccessToken = (tokens) => {
 
   try {
     const parsed = JSON.parse(tokens);
```

Comment:
Should probably add a type on this so we know the expected result.

Reply:
already added
<img width="874" alt="Screenshot 2024-07-24 at 17 20 38" src="https://github.com/user-attachments/assets/017d6307-e5d8-4133-b1e1-f2cb03b2db1c">

Reply:
I meant as `const parsed = JSON.parse(tokens) as {access_token: string,...};`

Reply:
I don't see the need because the function is typed, and we can see what it returns.